download-kafka.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-mirror=$(curl --stderr /dev/null https://www.apache.org/dyn/closer.cgi\?as_json\=1 | jq -r '.preferred')
+mirror="https://archive.apache.org/dist/"
```

Comment:
Why have you removed mirror selection?

Reply:
Actually because `0.10.0.0` doesn't exists there (Take a look at http://apache.spd.co.il/kafka/)

Reply:
`closer.cgi` just rotates through the various mirrors closest to you.  So if a particular mirror doesn't have the requisite version of kafka, then retrying should eventually result in a mirror that does have the version.

Reply:
To be true, it doesn't exists in my region but I see what you are talking about

Reply:
Perhaps there's a better technique / algorithm to employ here? Or just make it user overridable - such as EXPLICIT_KAFKA_DOWNLOAD_MIRROR or similar?